src/components/InteractiveArticle.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+import Helmet from 'react-helmet';
+import NotFound from 'components/NotFound';
+
+export default class InteractiveArticle extends FalcorController {
+  static getFalcorPathSets(params) {
+    return [
+      // Fetch article metadata
+      ['articlesBySlug', params.articleSlug, ['title', 'teaser', 'slug', 'image']],
+      // Fetch interactive article html/js/css
+      // For now we only use the html part, the js and css parts are for further improvements
+      ['articlesBySlug', params.articleSlug, 'interactiveData', ['html', 'js', 'css']],
+    ];
+  }
+
+  render() {
+    if (this.state.ready) {
+      if (!this.state.data) {
+        return <NotFound />;
+      }
+      const articleSlug = this.props.params.articleSlug;
+      // Access data fetched via Falcor
+      const articleData = this.state.data.articlesBySlug[articleSlug];
+      const interactiveCode = articleData.interactiveData;
+      // make sure article meta image has default
+      const articleMetaImage = articleData.image || 'https://thegazelle.s3.amazonaws.com/gazelle/2016/02/saadiyat-reflection.jpg';
+      const meta = [
+        // Search results
+        { name: 'description', content: this.props.teaser },
+
+        // Social media sharing
+        { property: 'og:title', content: `${articleData.title} | The Gazelle` },
+        { property: 'og:type', content: 'article' },
+        {
+          property: 'og:url',
+          content: `www.thegazelle.org/interactive/${articleData.slug}/`,
+        },
+        { property: 'og:image', content: articleMetaImage },
+        { property: 'og:image:width', content: '540' }, // 1.8:1 ratio
+        { property: 'og:image:height', content: '300' },
+        { property: 'og:description', content: articleData.teaser },
+        { property: 'og:site_name', content: 'The Gazelle' },
+      ];
+      const reactHtml = {
+        __html: interactiveCode.html,
+      };
+      return (
+        <div>
+          <Helmet
+            meta={meta}
+            title={`${articleData.title} | The Gazelle`}
+          />
+          <div dangerouslySetInnerHTML={reactHtml} />
+        </div>
+      );
+    }
+    return <div>loading...</div>;
```

Comment:
TODO: Make @zanemountcastle add a proper loading animation here (it's an interactive article so it should look nice right? And loading here could also be pretty relevant as there might be a lot of stuff loading).

Probably a good idea to wait with adding the code until I finished with my rebase though, I just don't want to forget this.